babel.config.js

```diff
@@ -53,13 +53,18 @@ module.exports = {
   presets: defaultPresets.concat(['@babel/preset-react']),
   plugins: [
     'babel-plugin-optimize-clsx',
+    `${__dirname}/packages/babel-plugin-set-component-name`,
```

Comment:
This wasn't needed for `babel-plugin-unwrap-createStyles`. My guess is that this isn't doing what it's supposed to. Just specify `babel-plugin-set-component-name` should do the trick.